shell/platform/embedder/embedder.h

```diff
@@ -306,12 +306,26 @@ typedef bool (*TextureFrameCallback)(void* /* user data */,
                                      FlutterOpenGLTexture* /* texture out */);
 typedef void (*VsyncCallback)(void* /* user data */, intptr_t /* baton */);
 
+typedef struct {
+  /// The width of the surface that will be backed by the fbo.
+  uint32_t width;
+  /// The height of the surface that will be backed by the fbo.
+  uint32_t height;
+} FlutterFrameInfo;
```

Comment:
This seems like something we can reuse later. How about `FlutterUIntSize`? There is already a `FlutterSize` but those are doubles.

Reply:
Do you foresee a case where this object will be need to expanded? If so, `FBOWithFrameInfoCallBack` will need to take a pointer to this. It will also need to have a struct size member.

Reply:
Done